docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc

```diff
@@ -333,8 +333,10 @@ Every query operation accepts passing parameters by index (`Object...`), or by n
 
 == The DAO/Repository option
 
-Look, we get it: you have a love/hate relationship with DAOs/Repositories but you can't live without them. We don't judge, we
-know life is tough and we've got you covered.
+Repositories can be amazing or evil, depending on the complexity of your needs. When they work, 
+they can save you from writing lots of boilerplate and error-prone JPA/SQL/Criteria queries, 
+but when they fail you need a way to drop back to more traditional methods... Don't worry, 
+Panache and Quarkus have you covered either way.
```

Comment:
The first sentence feels a bit long and still has some judgment. Maybe going straigth to the point should be bettre.

> Repository is a very popular design and can be very accurate for some use case, Panache and Quarkus have you covered ...

Reply:
I like it! Updated.

Reply:
Still thinking that the following sentence is not needed:
> but when you need to support a complex use case you can revert
to more traditional methods...

And the Don't worry seems to miss some introduction
> If you need them, don't worry ...